warehouse/query-core/src/test/java/datawave/query/jexl/DatawaveJexlEngineTest.java

```diff
@@ -0,0 +1,51 @@
+package datawave.query.jexl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.List;
+
+import org.apache.accumulo.core.data.Key;
+import org.apache.commons.jexl3.JexlException;
+import org.apache.commons.jexl3.internal.Script;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import datawave.query.attributes.Content;
+import datawave.query.attributes.Document;
+
+public class DatawaveJexlEngineTest {
+
+    private DatawaveJexlEngine engine = null;
+
+    @BeforeEach
+    public void setup() {
+        engine = ArithmeticJexlEngines.getEngine(new HitListArithmetic());
+    }
+
+    @Test
+    public void testScript() {
```

Comment:
please add a description and comments to outline the expected behavior of the test